lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -58,15 +63,21 @@ class FatMemManager : public OpSemMemManager {
   }
   /// \brief Creates a fat memory value from raw memory with default value for
   /// fat
-  FatMemValTy mkFatMem(MemValTy rawMem) const { return strct::mk(rawMem); }
+  FatMemValTy mkFatMem(MemValTy rawMem) const {
+    return strct::mk(rawMem, m_ctx.alu().si(0, slotBitWidth),
```

Comment:
type error. Slots of fat memory are memories (i.e., arrays). This works now because shadow memory is not supported.

Reply:
Ack